Jamfile

```diff
@@ -505,7 +505,7 @@ feature export-extra : off on : composite propagated ;
 variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
-	  <export-extra>on <debug-iterators>on <threading>multi
+	  <export-extra>on <debug-iterators>on <threading>multi <inlining>on
```

Comment:
I suppose that `boost-build` was upgraded then, it's `off` in master for that reason